src/os/bluestore/fastbmap_allocator_impl.cc

```diff
@@ -465,14 +477,27 @@ bool AllocatorLevel01Loose::_allocate_l1(uint64_t length,
           (idx * d1 + free_pos / L1_ENTRY_WIDTH + 1) * l0_w,
           allocated,
           res);
-        slot_val &= (~slot_t(L1_ENTRY_MASK)) << free_pos;
+
```

Comment:
I understand that L1_ENTRY_NOT_USED will never be written to l1 slotset.
Maybe there should be assert((free_pos & 1) == 0) @line470 to catch if bit combination "10" is used.